### PR TITLE
fixes and additions to working with time docs

### PR DESCRIPTION
**Pull request by jimtng**

### CODE REVIEW

File: docs/usage/misc/time.md [hunk 1]
```diff
@@ -125,6 +186,16 @@ else
   logger.info("Not in time range")
 end
 
-(java.time.Month::NOVEMBER..java.time.Month::DECEMBER).cover?(Time.now)
-(java.time.Month::NOVEMBER..java.time.Month::DECEMBER).cover?(Date.today)
+java_import java.time.Month # This is already imported in OpenHAB 3.4
```
Comment (ccutrer) [inline docs/usage/misc/time.md:189]: and in 3.3, by our library

File: docs/usage/misc/time.md [hunk 2]
```diff
@@ -125,6 +186,16 @@ else
...
+
+# Compare against Month
+Time.now.between?(Month::NOVEMBER..Month::DECEMBER)
+Date.today.between?(Month::NOVEMBER..Month::DECEMBER)
+ZonedDateTime.now.between?(Month::NOVEMBER..Month::DECEMBER)
+
+# Compare against MonthDay
+ZonedDateTime.now.between?("05-01".."12-01")
```
Comment (ccutrer) [inline docs/usage/misc/time.md:197]: again. now that we've shown they all work, we should be pushing towards the shortest version - `Time.now..between?`

**Review by ccutrer — CHANGES_REQUESTED** [review]

File: docs/examples/how_do_i.md [hunk 3]
```diff
@@ -743,7 +743,7 @@ end
 
 ```ruby
 # The range can cross midnight
-if between('10pm'..'5am').cover?(LocalTime.now)
+if LocalTime.now.between?('10pm'..'5am')
```
[the remark this answers is not in the document]
Comment (jimtng) [reply docs/examples/how_do_i.md]: done

File: docs/usage/misc/time.md (hunk 1 above)
Comment (jimtng) [reply docs/usage/misc/time.md]: I've removed it completely now.

File: docs/usage/misc/time.md (hunk 2 above)
Comment (jimtng) [reply docs/usage/misc/time.md]: good idea

**Review by ccutrer — APPROVED** [review]

**Pull request merged**